inference-engine/tests/functional/plugin/shared/include/behavior/core_integration.hpp

```diff
@@ -453,6 +453,12 @@ TEST_P(IEClassBasicTestP, ImportNetworkWithNullContextThrows) {
 // LoadNetwork
 //
 
+TEST_P(IEClassNetworkTestP, LoadNetworkWithoutDeviceNameNoThrow) {
+  SKIP_IF_CURRENT_TEST_IS_DISABLED()
+  Core ie;
+  ASSERT_NO_THROW(ie.LoadNetwork(actualNetwork));
```

Comment:
currently, AUTO selects between GPU and CPU, how will it work on yocto?

Reply:
I guess that will throw exception, will the yocto case run this test? which plugin will the yocto run? (kmb-plugin?) not very clear with that usecase.. @wangleis

Reply:
In this case `LoadNetwork` with no args does not work on all platforms.

Reply:
I see tests on cpuFuncTest and gpuFuncTest are passed, so what do you mean not workable on all platforms, could you please provide more info?